flocker/provision/_aws.py

```diff
@@ -58,6 +58,13 @@
 INSTANCE_TIMEOUT = 300
 
 
+class FailedToRun(Exception):
+    """
+    Raised if a pending AWS failed to become running.
+    See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-lifecycle.html
+    """     # noqa
```

Comment:
Yeah, but I do not want to break the lint build.
Perhaps I could remove the URL, it does not seem to be that informative to me now (in the context of the exception).